src/Microsoft.Health.Dicom.Functions/Duplicate/Models/DuplicateBatchArguments.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Functions.Duplicate.Models;
+
+/// <summary>
+///  Represents input to <see cref="DuplicateDurableFunction.DuplicateBatchAsync"/>
+/// </summary>
+public sealed class DuplicateBatchArguments
+{
+    /// <summary>
+    /// Gets or sets the number of threads available for each batch.
+    /// </summary>
+    public int ThreadCount { get; } = 5;
```

Comment:
In retrospect, we probably can ignore this value too, as it doesn't impact the activity input or output. I think I was overzealous before for re-index

Reply:
So you may not need this entire class